samples/python/lk_track.py

```diff
@@ -1,106 +0,0 @@
-#!/usr/bin/env python
```

Comment:
Move to snippets for calcOpticalFlowPyrLK.

Reply:
@asmorkalov corresponding cpp sample [lkdemo.cpp](https://github.com/opencv/opencv/blob/5.x/samples/cpp/lkdemo.cpp ) is not in snippets. Should we still move python or keep it same as cpp